uwblueprint/community-fridge-kw
# feat: admin create checkin page

## Brief description. What is this change? 

### [Admin Create Checkin Page](https://www.notion.so/uwblueprintexecs/Admin-Create-Checkin-Page-2d055ff938a4475198542488eaf320e6)



## Implementation description. How did you make this change?
* added page at `/admin/create-checkins` endpoint



## Steps to test
1. Create an admin account
2. Login and go to the `/admin/create-checkins` endpoint
3. Create a set of check-ins
4. Verify in Postman that the correct check-ins were created
5. Try creating a check-in where the start time is after the end time (do this in the **select time range** section)
6. Verify that you do not get redirected to a new page and that you can see an appropriate error state (for this case, the message is set to be "End time cannot be before start time")

**TO NOTE**: 
* you currently CANNOT create check-ins where the start time is in the afternoon/evening and the end time is in the morning (e.g. 11:00 pm to 2:00 am); latest end time is midnight
* default values for date range are the current day and the day after the current day


![Screen Shot 2022-03-31 at 10 50 15 PM](https://user-images.githubusercontent.com/38363056/161185704-dcf188ac-9570-4498-bd10-0779e5178d79.png)
![Screen Shot 2022-03-31 at 10 51 54 PM](https://user-images.githubusercontent.com/38363056/161185888-b5d8907c-246b-44ef-8f93-7abce82296cd.png)



## Checklist
